src/Haskell/Ide/Engine/LSP/Reactor.hs

```diff
@@ -124,15 +130,16 @@ cancelRequest lid =
 -- | Execute multiple ide requests sequentially
 makeRequests
   :: [IdeDeferM (IdeResult a)] -- ^ The requests to make
+  -> String
   -> TrackingNumber
   -> J.LspId
   -> ([a] -> R ())          -- ^ Callback with the request inputs and results
   -> R ()
 makeRequests = go []
  where
-  go acc [] _ _ callback = callback acc
-  go acc (x : xs) tn reqId callback =
-    let reqCallback result = go (acc ++ [result]) xs tn reqId callback
-    in  makeRequest $ IReq tn reqId reqCallback x
+  go acc [] _ _ _ callback = callback acc
+  go acc (x : xs) d tn reqId callback =
+    let reqCallback result = go (acc ++ [result]) xs d tn reqId callback
```

Comment:
Isn't the order being reversed here?

Reply:
I don't think that changed on this PR? Is that a general point?